btsieve/tests/test_data/lib.sh

```diff
@@ -0,0 +1,58 @@
+container="" # Set by docker_run
+
+rm_file_if_exists() {
+    local file=$1
+
+    if [ -f $file ] ; then
+        rm $file
+    fi
+}
+
+docker_run() {
+    container=$(docker run -d -v /tmp/bitcoin:/root/.bitcoin coblox/bitcoin-core -regtest)
+    echo "Create container $container ... sleeping"
+    sleep 2
+}
+
+docker_stop() {
+    docker stop $container > /dev/null
+    echo "Stopped container $container"
+}
+
+docker_rm() {
+    docker rm $container > /dev/null
+    echo "Removed container $container"
+}
+
+docker_start() {
+    docker start $container > /dev/null
+    echo "Started container $container ... sleeping"
+    sleep 2
+}
+
+
+generate_101_blocks() {
+    docker exec $container bitcoin-cli -regtest generate 101 > /dev/null
+    echo "Generated 101 blocks"
+}
+
+generate_block() {
```

Comment:
```suggestion
generate_block() {
    if [ $# -ne 1 ]; then
        echo "Incorrect number of parameters on generate_block"
        exit 2
    fi
```
That can be helpful to avoid issues